test/kata6.joinNames.test.js

```diff
@@ -2,4 +2,20 @@ const { joinNames } = require("../src");
 
 describe("joinNames", () => {
   test("returns string of names, seperated by commas and an ampersand", () => {});
+  expect(
+    joinNames([{ name: "Bart" }, { name: "Lisa" }, { name: "Maggie" }])
+  ).toEqual("Bart, Lisa & Maggie");
+
+  expect(
+    joinNames([
+      { name: "Peter Griffin" },
+      { name: "Stewie Griffin" },
+      { name: "Lois Griffin" },
+      { name: "Meg Griffin" },
+      { name: "Brian Griffin" },
+      { name: "Chris Griffin" },
+    ])
+  ).toEqual(
+    "Peter Griffin, Stewie Griffin, Lois Griffin, Meg Griffin, Brian Griffin & Chris Griffin"
+  );
```

Comment:
This should be updated, at the moment all your assertions are not being wrapped by a test block as they should be:
```
  it("returns string of names, seperated by commas and an ampersand", () => {
    expect(
      joinNames([{ name: "Bart" }, { name: "Lisa" }, { name: "Maggie" }])
    ).toEqual("Bart, Lisa & Maggie");

    expect(
      joinNames([
        { name: "Peter Griffin" },
        { name: "Stewie Griffin" },
        { name: "Lois Griffin" },
        { name: "Meg Griffin" },
        { name: "Brian Griffin" },
        { name: "Chris Griffin" },
      ])
    ).toEqual(
      "Peter Griffin, Stewie Griffin, Lois Griffin, Meg Griffin, Brian Griffin & Chris Griffin"
    );
  });
```
It would be great to add another test case for a single name, since you have code that covers that scenario it makes sense for it to be tested in the same way